config/functions

```diff
@@ -318,6 +356,36 @@ listremoveitem() {
   echo "${tmp_array[@]}"
 }
 
+# check if a flag is enabled
+# $1: flag-name, $2: default (yes/no), $3: ingenious check (none,only-disable,only-enable)
+# set variable [flag]_ENABLED=(yes/no)
+# return 0 is flag is enabled, otherwise 2
+flag_enabled() {
+  # make flag name upper case and replace hyphen with underscore, to use as variable name
+  local flag_name=${1^^}
+  flag_name="${flag_name/[:-]/_}"
```

Comment:
Can you change this to be:
```
flag_name="${flag_name//[:-]/_}"
```
otherwise only the first `:` or `-` will be replaced.